docs/source/reference/jinja.rst

```diff
@@ -1,101 +1,87 @@
-| ``version_less_than``          | Returns if LHS version is lesser than RHS                      |
-|                                | version. Both input has to follow semantic                     |
-|                                | version syntax. E.g. {{"1.6.0" | version_less_than("1.7.0")}}. |
-+--------------------------------+----------------------------------------------------------------+
-| ``version_equal_than``         | Returns if LHS version is equal to RHS version.                |
-+--------------------------------+----------------------------------------------------------------+
-| ``version_bump_major``         | Bumps the major version and returns new                        |
-|                                | version.                                                       |
-+--------------------------------+----------------------------------------------------------------+
-| ``version_bump_minor``         | Bumps the minor version and returns new                        |
-|                                | version.                                                       |
-+--------------------------------+----------------------------------------------------------------+
-
-Examples of how to use filters are available in
-:github_st2:`st2/contrib/examples/actions/chains/data_jinja_filter.yaml </contrib/examples/actions/chains/data_jinja_filter.yaml>`,
-:github_st2:`st2/contrib/examples/actions/chains/time_jinja_filter.yaml </contrib/examples/actions/chains/time_jinja_filter.yaml>`,
-:github_st2:`st2/contrib/examples/actions/chains/data_jinja_filter.yaml </contrib/examples/actions/chains/decrypt_kv_jinja_filter.yaml>`,
-and :github_st2:`st2/contrib/examples/actions/chains/version_jinja_filter.yaml </contrib/examples/actions/chains/version_jinja_filter.yaml>`.
-
-
-|st2| supports `Jinja2 variable templating <http://jinja.pocoo.org/docs/dev/templates/#variables>`__
-in Rules, Action Chains and Actions etc. Jinja2 templates support `filters <http://jinja.pocoo.org/docs/dev/templates/#list-of-builtin-filters>`__ to allow some advanced capabilities in working with variables. |st2| has further
-added some more filters.
+For brevity, only simple Jinja patterns for each filter are documented below. "Real-world" usage
+will depend on the type of content where the filters are being applied (sensors, triggers, rules,
+action and workflows) and their syntax. More detailed examples can be found in several of the
+ActionChains located in the ``examples`` pack:
+:github_st2:`st2/contrib/examples/actions/chains/ </contrib/examples/actions/chains/>`.
 
-.. _jinja-jinja-filters:
+..  TODO We should consider separating each specific usage into individual ActionChains and refer to
+    it using literalinclude (i.e. .. literalinclude:: /../../st2/contrib/examples/actions/workflows/mistral-jinja-branching.yaml)
+    so we can just use the code as the source of truth. Then, we can remove the above note.
```

Comment:
Cool. To keep things easy, we'd have to split up the existing chains into one example per workflow (not to mention some of the filters just outright don't have examples) so I'm thinking we just leave the TODO in here for now until the examples pack is updated. Opened https://github.com/StackStorm/st2/issues/3553 to track this